src/pythainer/builders/utils.py

```diff
@@ -243,3 +243,87 @@ def project_git_cmake_build_install(
         install=install,
         cleanup=cleanup,
     )
+
+def remove_files(
+        builder: PartialDockerBuilder,
+        file_path: PathType
+        ) -> None:
+    """
+    Remove the file
+    """
+    if Path.exists(file_path):
+        builder.run(
+            command=f"rm -f {file_path}"
+        )
+
+def wget_from_url(
+        builder: PartialDockerBuilder,
+        package_name: str,
+        package_url: str,
+        package_savepath: PathType = Path("/tmp")
+) -> None:
+    """
+    Download package with wget command from the input address
+    """
+    builder.run(
+        command=f"wget -qO {package_savepath / package_name}.deb {package_url}"
```

Comment:
same comment as above. Perhaps you can return just the string of the command and add it to the list?